core/nuxeo-resource.js

```diff
@@ -310,10 +310,14 @@ import './nuxeo-connection.js';
         .execute(options)
         .then((response) =>
           response.text().then((text) => {
-            try {
-              return text ? JSON.parse(text) : {};
-            } catch (e) {
-              return { error: 'Invalid json' };
+            if (this.headers.accept.includes('application/json')) {
```

Comment:
If I understood @nelsonsilva 's comment correctly, we should not match the request headers but the response content type. 🤔